pkg/queue/cluster_queue_strict_fifo_test.go

```diff
@@ -34,11 +35,14 @@ const (
 )
 
 func TestFIFOClusterQueue(t *testing.T) {
-	q, err := newClusterQueue(&kueue.ClusterQueue{
-		Spec: kueue.ClusterQueueSpec{
-			QueueingStrategy: kueue.StrictFIFO,
+	q, err := newClusterQueue(
+		&kueue.ClusterQueue{
```

Comment:
nit: revert formatting changes

Reply:
Reverting, note: after running `go fmt` this comes out as:

```go
	q, err := newClusterQueue(&kueue.ClusterQueue{
		Spec: kueue.ClusterQueueSpec{
			QueueingStrategy: kueue.StrictFIFO,
		},
	},
		workload.Ordering{
			PodsReadyRequeuingTimestamp: config.EvictionTimestamp,
		})
```

Reply:
Let's leave the PR as-is